### PR TITLE
31 sign in

### LINKED ISSUE #31: sign in
As a User
So I can list spaces and book spaces
I want to sign in

### PR DESCRIPTION
**Pull request by tobenna**
Closes #31

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -17,5 +17,7 @@
 require_relative './controllers/spaces'
 require_relative './controllers/users'
 require_relative './controllers/requests'
+require_relative './controllers/requests'
```
Comment (kevinpmcc) [inline app/app.rb]: duplicate requrie

File: spec/features/helpers/method_helper.rb [hunk 2]
```diff
@@ -12,3 +12,12 @@ def sign_up
   fill_in(:password_confirmation, with: '123456')
   click_button('Sign Up')
 end
+
+def create_user(email: 'adil@dil.com')
```
Comment (adilw3nomad) [inline spec/features/helpers/method_helper.rb]: we could use factory girl to insert records into database

**Comment by adilw3nomad** [comment]
:shipit:

File: spec/features/helpers/method_helper.rb (hunk 2 above)
Comment (tobenna) [inline spec/features/helpers/method_helper.rb]: :+1:

**Comment by kevinpmcc** [comment]
:shipit:

**Pull request merged**